share/task.yml

```diff
@@ -22,7 +22,8 @@
   gather_facts: True
 
   vars:
-    - repo: https://github.com/edx/edx-analytics-pipeline.git
+#    - repo: https://github.com/edx/edx-analytics-pipeline.git
```

Comment:
We shouldn't comment out code.

Regardless of the cruft that accumulates,
it leads to false positives when searching the codebase.

A relevant example is with the removal/commenting of the `update pip` task earlier today.
Because that string was still in the codebase, instead of being deleted,
it kept showing up when `grep`ing the codebase which meant we had to parse through the results and determine which were valid and which weren't. Had the code been removed, it would have been easier to just realize which were positive hits.

And since the change is recorded in source control,
if we need to update/undo/audit the changes later,
it's still as easy as a `git revert`.